backend/fleet_management/migrations/0005_passenger_email.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.2 on 2018-12-08 11:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fleet_management', '0004_auto_20181129_2056'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='passenger',
+            name='email',
+            field=models.EmailField(default='email@email.com', max_length=254),
```

Comment:
I guess `email@email.com` is an example data not a valid value which should be in database. If I'm right please change that.